environment.yml

```diff
@@ -22,3 +22,6 @@ dependencies:
   - biopet-vcfstats==1.2
   - fontconfig==2.13.0
   - gxx_linux-64==7.2.0=h550dcbe_27
+  - tabix==0.2.6
+  - biopet-bamstats==1.0
```

Comment:
Please use bamstats==1.0.1. Bamstats will now crash in the beginning if it cannot write the inputfile, instead of the end (which may take some time)